src/main/resources/templates/albums.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="'en" xmlns:th="http://www.w3.org/1999/xhtml">

```

Comment:
When using Thymeleaf, all the HTML templates need this line of code instead of what you have. <html xmlns:th="http://www.thymeleaf.org" lang="'en">